packages/eslint-config-react/index.js

```diff
@@ -1,269 +1,352 @@
 /**
  * strvcom/eslint-config-react
  *
- * @author      Robert Rossmann <rr.rossmann@me.com>
+ * @author      Pavel Prichodko <pavel.prichodko@strv.com>
  * @copyright   2019 STRV
  * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
  */
 
 'use strict'
 
 module.exports = {
+  root: true,
+  extends: ['@strv/eslint-config-base'],
 
-  extends: require.resolve('@strv/eslint-config-base'),
+  parser: 'babel-eslint',
+
+  plugins: ['react', 'jsx-a11y', 'react-hooks'],
 
   env: {
-    es6: true,
     browser: true,
+    commonjs: true,
+    es6: true,
+    jest: true,
+    node: true,
```

Comment:
Because of process.env.variables and webpack :) 
https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables

And this little guy:
![image](https://user-images.githubusercontent.com/2115953/55627758-40d36580-57af-11e9-83de-e0b129803648.png)

Would you recommend to go with `globals` instead?